icu4c/source/tools/genrb/filterrb.h

```diff
@@ -0,0 +1,88 @@
+// © 2018 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef __FILTERRB_H__
+#define __FILTERRB_H__
+
+#include <list>
+#include <string>
+#include <map>
+#include <ostream>
+
+#include "unicode/utypes.h"
+
+
+class ResKeyPath {
+  public:
```

Comment:
I am pretty sure that we don't indent public: and private: labels.

Reply:
It's a "half-indent".  This is Google style.  In ICU4C, it seems like we're inconsistent, but non-indented seems to win.  I'll change it.